src/libs/styles.c

```diff
@@ -218,7 +219,11 @@ static void _styles_row_activated_callback(GtkTreeView *view, GtkTreePath *path,
   gtk_tree_model_get(model, &iter, DT_STYLES_COL_FULLNAME, &name, -1);
 
   const GList *list = dt_view_get_images_to_act_on(TRUE, TRUE, FALSE);
-  if(name) dt_styles_apply_to_list(name, list, gtk_toggle_button_get_active(GTK_TOGGLE_BUTTON(d->duplicate)));
+  if(name)
+  {
+    dt_styles_apply_to_list(name, list, gtk_toggle_button_get_active(GTK_TOGGLE_BUTTON(d->duplicate)));
+    g_free(name);
```

Comment:
Nice one, I would have sworen that the returned value was not a copy of the in-tree value !